guava/src/com/google/common/hash/AbstractHashFunction.java

```diff
@@ -33,14 +37,17 @@
     return newHasher().putObject(instance, funnel).hash();
   }
 
+  @SuppressWarnings({"lowerbound:argument.type.incompatible",// Since len is length of `input` with min length of 1,
+          // len * 2 can't be negative
```

Comment:
The checker should definitely infer this. If it doesn't, that's a bug. `CharSequence#length` should return a non-negative regardless of the MinLen, and the LBC should (does) infer that a constant positive times a non-negative is also non-negative.

Reply:
Either way the minimum length isn't relevant to the code's correctness.